pygluu/kubernetes/settings.py

```diff
@@ -142,38 +142,38 @@ def is_exist(self):
         else:
             return True
 
-    def validate(self):
```

Comment:
How is validation occurring? Helm does validation automatically so that should be picked and the installer should allow to fix the issue and retry.

Reply:
remove commented lines. Not a good practice.

Reply:
Im not sure what happened here. I see the validation commented out?